src/ResourceManager/Network/Commands.Network/help/Reset-AzureRmVirtualNetworkGateway.md

```diff
@@ -18,8 +18,13 @@ Reset-AzureRmVirtualNetworkGateway -VirtualNetworkGateway <PSVirtualNetworkGatew
 ```
 
 ## DESCRIPTION
-
+Resets the Virtual Network Gateway
 ## EXAMPLES
+### Example 1:
+```
+$Gateway = Get-AzureRmVirtualNetworkGateway -Name "ContosoVirtualGateway"
+Reset-AzureRmVirtualNetworkGateway -VirtualNetworkGateway $Gateway
```

Comment:
Can you simply pipe between the cmdlets?

Get-AzureRmVirtualNetworkGateway -Name "FContosoVirtualGateway" | Reset-AzureRmVirtualNetworkGateway